### DISCUSSION TITLE
Migration getting registered but not executed? Can't ALTER column to be nullable.

### DISCUSSION THREAD

**Discussion by hsein-bitar** [planetscale/discussion#626] [Bug Reports]
## Context
- I am working on a Laravel App connected to a PlanetScale DB
- Goal: running a migration to alter a `varchar(255)` column to be nullable
- If I do `php artisan migrate --pretend`, the SQL output is `ALTER TABLE scorecards CHANGE description description VARCHAR(255) DEFAULT NULL`
- Running `php artisan migrate -vvv` adds the migration to the migrations table in PlanetScale, but the column type remains non-nullable, the command also exits with code 0 and without any errors.
- Running the raw SQL output in the PlanetScale Web Console does change the column to be nullable, this is the desired outcome.
- On a local MySQL db, the migration does update the column to be nullable ( issue only happening on PlanetScale DB afaik )

So something is not going as expected, somewhere between registering the migration, and executing it :/
I appreciate any help.

## References
The migration is pretty simple, shown below:
```
return new class extends Migration
{
    /**
     * Run the migrations.
     */
    public function up(): void
    {
        Schema::table('scorecards', function (Blueprint $table) {
            $table->string('description')->nullable()->change();
        });
    }

    /**
     * Reverse the migrations.
     */
    public function down(): void
    {
        Schema::table('scorecards', function (Blueprint $table) {
            $table->string('description')->nullable(false)->change();
        });
    }
};
```
Running the migration in the server results in no errors, but does register a migration in migrations table!
![image](https://github.com/planetscale/discussion/assets/49834340/eea03644-43ca-4bc3-8258-433b260e3203)

## Note
I also can't find a trace of executing the ALTER query in the PlanetScale dashboard

## Update #1
I discovered that running the raw SQL output from the server executes the expected change, this means that Laravel itself, is sending the SQL to insert a migration record, but it's not sending the ALTER query ( I think )

**Comment by mscoutermarsh** [comment]
Is it possible that the migration is already recorded as having run in the migration table? Causing Laravel not to actually run it?

Other than that, I can't think of any reason why this wouldn't be running.
